bbb-common-message/src/main/scala/org/bigbluebutton/common2/msgs/PresentationPodsMsgs.scala

```diff
@@ -228,6 +230,7 @@ case class PresentationConversionRequestReceivedEventMsgBody(
     current:        Boolean,
     presName:       String,
     downloadable:   Boolean,
+    removable:      Boolean,
```

Comment:
These events are also used by bbb-html5, see https://github.com/bigbluebutton/bigbluebutton/blob/v2.5.x-release/bigbluebutton-html5/imports/api/presentations/server/eventHandlers.js 

I get

```
I20220203-03:03:59.714(0)? info: Updated presentation conversion status=SUPPORTED_DOCUMENT id=d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1643857434602 meeting=1c6637a8f2e1f75e06ff9984894d6bd16a3a36a9-1643857434601
I20220203-03:04:00.408(0)? error: PresentationConversionCompletedEvtMsg: Error: Match error: Unknown key in field removable
I20220203-03:04:00.409(0)?     at check (packages/check/match.js:36:17)
I20220203-03:04:00.409(0)?     at addPresentation (imports/api/presentations/server/modifiers/addPresentation.js:32:3)
I20220203-03:04:00.409(0)?     at EventEmitter.handlePresentationAdded (imports/api/presentations/server/handlers/presentationAdded.js:13:10)
I20220203-03:04:00.409(0)?     at EventEmitter.emitAsync (/home/bigbluebutton/src/bigbluebutton-html5/node_modules/eventemitter2/lib/eventemitter2.js:463:31)
I20220203-03:04:00.409(0)?     at MeetingMessageQueue.handleTask (imports/startup/server/redis.js:105:10)
I20220203-03:04:00.409(0)?     at imports/startup/server/redis.js:117:28
I20220203-03:04:00.409(0)?     at Queue.start (/home/bigbluebutton/src/bigbluebutton-html5/node_modules/queue/index.js:145:17)
I20220203-03:04:00.410(0)?     at Queue.<computed> [as push] (/home/bigbluebutton/src/bigbluebutton-html5/node_modules/queue/index.js:59:12)
I20220203-03:04:00.410(0)?     at MeetingMessageQueue.add (imports/startup/server/redis.js:116:16)
I20220203-03:04:00.410(0)?     at RedisPubSub.handleMessage (imports/startup/server/redis.js:318:63)
I20220203-03:04:00.410(0)?     at runWithEnvironment (packages/meteor.js:1320:24)
```

not matching object structure. Please commit the files you had to modify in bbb-html5 too

Reply:
Hey @antobinary , sorry about it, that passed without me noticing it, I've just made a commit with the other changes, apparently there is some conflict which I am going to tackle right now.